deepchem/models/losses.py

```diff
@@ -41,6 +41,23 @@ def _create_pytorch_loss(self):
     return torch.nn.L1Loss(reduction='none')
 
 
+class HuberLoss(Loss):
```

Comment:
Could you add the new loss to the docs? 

https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/models.rst#losses

Reply:
@rbharath  Thank you! I have made the change.